src/index.ts

```diff
@@ -0,0 +1,30 @@
+// core
+export { DOMAPI, htmlDomApi } from './htmldomapi';
+export { init } from './init';
+export { ThunkData, Thunk, ThunkFn, thunk } from './thunk';
+export { Key, VNode, VNodeData, vnode } from './vnode';
+
+// helpers
+export { AttachData, attachTo } from './helpers/attachto';
+export { array, primitive } from './is';
+export { toVNode } from './tovnode';
+export {
+  VNodes,
+  VNodeChildElement,
+  ArrayOrElement,
+  VNodeChildren,
+  h,
+} from './h';
+
+// types
+export * from './hooks';
+export { Module } from './modules/module';
+
+// modules
+export { Attrs, attributesModule } from './modules/attributes';
+export { Classes, classModule } from './modules/class';
+export { Dataset, datasetModule } from './modules/dataset';
+export { On, eventListenersModule } from './modules/eventlisteners';
+export { Hero, heroModule } from './modules/hero';
```

Comment:
let's kill this thing immediately. Will make the module a little smaller and also resolve https://github.com/snabbdom/snabbdom/issues/517

Reply:
Wanted to do that in a seperate PR

Reply:
I guess that's fine, since we're not actually pushing this out to a new npm version anyway. Just figured it's a little silly to carry forward this piece we know we don't want. but it's not a big deal.